### PR TITLE
admin list categories

### PR DESCRIPTION
**Pull request by khaledkz**
admin list categories[without bootstrap style]

### CODE REVIEW

**Review by kabaros — CHANGES_REQUESTED** [review]
if you're using the bootstrap from the CDN then remove it from package.json (and all the files in public) - the reason the style is not working is probably because you didn't wrap it in a container and such.

Also don't copy the files in each handlebars - just use a layout handlebar as we did in the projects before.

**Comment by khaledkz** [comment]
done

File: server/routes/admin.js [hunk 1]
```diff
@@ -2,12 +2,22 @@ var express = require('express');
 var router = express.Router();
 const categoryClient = require('../dbClients/categoriesDB');
 const articleClient = require('../dbClients/articlesDB')
+const categoryDB = require("../dbClients/categoriesDB");
```
Comment (kabaros) [inline server/routes/admin.js]: why are you importing categoryDB - you can just use categoryClient (it's the same module).

**Review by kabaros — CHANGES_REQUESTED** [review]

File: server/views/index.hbs [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+  <head>
+    <title>{{title}}</title>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">    <link rel='stylesheet' href='/stylesheets/style.css' />
+  </head>
+  <body>
+
 <h1>{{title}}</h1>
 <p>Welcome to {{title}}</p>
+ <!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
```
Comment (kabaros) [inline server/views/index.hbs]: you don't need stylesheets again here - they come from layout

File: server/routes/admin.js (hunk 1 above)
Comment (kabaros) [reply server/routes/admin.js]: @kharazzoro did you read this comment

File: server/views/index.hbs (hunk 2 above)
Comment (kabaros) [reply server/views/index.hbs]: @kharazzoro

Comment (kabaros) [reply server/views/index.hbs]: this comment @kharazzoro

**Pull request merged**